tests/testthat/test-cleaners.R

```diff
@@ -1,62 +1,78 @@
 test_that("cleaners work", {
   # sites to use for testing
-  college <- read_wiki_raw("https://en.wikipedia.org/wiki/List_of_colleges_and_universities_in_Massachusetts")
+  college <- read_wikitables("https://en.wikipedia.org/wiki/List_of_colleges_and_universities_in_Massachusetts")
 
-  presidents <- read_wiki_raw("https://en.wikipedia.org/wiki/List_of_presidents_of_the_United_States_by_age")
+  presidents <- read_wikitables("https://en.wikipedia.org/wiki/List_of_presidents_of_the_United_States_by_age")
 
-  marvel <- read_wiki_raw("https://en.wikipedia.org/wiki/List_of_Marvel_Cinematic_Universe_films")
+  marvel <- read_wikitables("https://en.wikipedia.org/wiki/List_of_Marvel_Cinematic_Universe_films")
 
-  ascii <- read_wiki_raw("https://en.wikipedia.org/wiki/ASCII")
+  ascii <- read_wikitables("https://en.wikipedia.org/wiki/ASCII")
 
   # test clean_wiki_names()
   # test that column names clean as expected
-  vars <- c("school", "location", "control", "type", "enrollment_2013_2014", "founded", "accreditation")
-  clean_wiki <- clean_wiki_names(college)
-  expect_equal(names(clean_wiki), vars)
+  vars <- c("school", "location", "control", "type", "enrollment", "founded", "accreditation")
+  clean_college <- clean_wiki_names(college)
+  clean_college1 <- clean_college %>% purrr::pluck(1)
+  expect_equal(names(clean_college1), vars)
 
   # test that correctly passes arguments to janitor::clean_names()
   expect_equal(
-    names(clean_wiki_names(college, "all_caps")),
+    names(clean_wiki_names(college, "all_caps") %>% purrr::pluck(1)),
     toupper(vars)
   )
 
   # test that footnotes are removed
-  expect_true(all(!stringr::str_detect(names(clean_wiki), "\\[")))
+  expect_true(all(!stringr::str_detect(names(clean_college1), "\\[")))
 
   # test that removes columns without name -- note: need to do
   # remove footnotes before this works
+  ## NOTE-- remove footnotes not working for this
   ascii_clean <- ascii %>%
-    remove_footnotes() %>%
-    clean_wiki_names()
-  expect_lt(ncol(ascii_clean), ncol(ascii))
+    #remove_footnotes() %>%
+    clean_wiki_names() %>%
+    purrr::pluck(1)
+  expect_lt(ncol(ascii_clean), ncol(ascii %>% purrr::pluck(1)))
 
   # test that removes special characters in names
+  ## clean_wiki_names won't work with dummy data-- not sure why
   dummy_data_1 <- tibble::tribble(
     ~`first@`, ~second, ~third,
     "?", "two", "three",
     "_", "five", 7,
     "N/A", "ten", "eleven"
   )
 
-  expect_false(clean_wiki_names(dummy_data_1)[,1] %>%
+  expect_false(clean_wiki_names(dummy_data_1) %>%
+                 purrr::pluck(1) %>%
```

Comment:
If you comment out this line, does it work?

Reply:
I got this one to work by making the dummy data a list.